code/datums/wounds/cranial_fissure.dm

```diff
@@ -115,9 +117,49 @@
 
 	return TRUE
 
+/datum/wound/cranial_fissure/treat(obj/item/I, mob/user)
+	. = ..()
+	if(user.zone_selected != BODY_ZONE_HEAD)
+		return FALSE
+
+	if(istype(I, /obj/item/kitchen/spoon))
```

Comment:
We shouldn't be checking the item here at all, it should be handled by ``item_can_treat``, this seems to be for special effects from treating with a specific item, like if you use my previous suggestion you'd have to move the brain into the bag in this proc.